server/src/models/characterModel.ts

```diff
@@ -174,15 +248,15 @@ const characterSchema = new Schema({
       },
     },
   },
-  speeds: {
+  speed: {
     walking: {
       type: Number,
       default: 35,
```

Comment:
default 30?

Reply:
that was me just testing out the math function, its 30 now